js-exercises/bind-method/bindMethod.js

```diff
@@ -0,0 +1,8 @@
+// You can change the `args`
+function bind(fn, obj) {
+  return function binder(...args) {
+    return fn.apply(obj, args);
```

Comment:
use spread operator
this will fail  for this test case
```
test('test rest is used or not', () => {
    const foo = function (x, z) {
      return this.y + x + z;
    };
    const obj = { y: 10 };
    var y = 100;
    const bar = bind(foo, obj);
    expect(bar(10, 20)).toBe(40);
  });
```